libs/langchain/langchain/chains/sql_database/query.py

```diff
@@ -116,7 +116,8 @@ def create_sql_query_chain(
         prompt_to_use = SQL_PROMPTS[db.dialect]
     else:
         prompt_to_use = PROMPT
-    if {"input", "top_k", "table_info"}.difference(prompt_to_use.input_variables):
+    if {"input", "top_k", "table_info"}.difference(prompt_to_use.input_variables +
+                                                   list(prompt_to_use.partial_variables)):
```

Comment:
I used the same way as agents https://github.com/langchain-ai/langchain/blob/9aabb446c597292c0b387779582795d2e28feb28/libs/langchain/langchain/agents/openai_tools/base.py#L81-L83